examples/wordpress/wordpress.jsonnet

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2018 Bitnami
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+// Example to demonstrate kubecfg using kube-libsonnet
+// This should not necessarily be considered a model jsonnet example
```

Comment:
Can probably drop this disclaimer now.
I added it originally to kubecfg's own guestbook example - because I didn't want to deal with questions about why I wasn't using ksonnet-lib.  I don't think that concern applies to jsonnet in kube-libsonnet.